js/pages/characterizations/components/feature-analyses/feature-analyses-browser.less

```diff
@@ -8,6 +8,15 @@
   .fe-btn {
     display: flex;
   }
+  &__button-panel {
+    display: flex;
+    flex-direction: row-reverse;
+    padding: 1rem 2rem;
+
+    .btn {
```

Comment:
Same question as above. Plus, it looks like code duplication (`button-panel`)

Reply:
fixed